agent.go

```diff
@@ -200,6 +205,21 @@ func (c *container) getProcess(execID string) (*process, error) {
 	return proc, nil
 }
 
+func (s *sandbox) getSandboxStorages() []sandboxStorage {
+	return s.storages
+}
+
+func (s *sandbox) setSandboxStorages(sb sandboxStorage) {
```

Comment:
nit: `sb` can be confusing with `sandbox` :(

Reply:
yes, better to rename this, something like `sbStorage`

Reply:
I would also rename the method itself to something like appendSandboxStorage or even setSandboxStorage to make it clear we are adding this storage to existing list.